.storybook/theme.js

```diff
@@ -0,0 +1,9 @@
+import { create } from '@storybook/theming';
+import Logo from './Logo.png';
+
+export default create({
+  base: 'light',
+  brandTitle: 'Factiverse Design System',
+  brandUrl: 'http://factiverse.no/',
```

Comment:
This should be the factiverse storybook URL 

`https://factiverse.github.io/storybook-design-system/?path=/story/base-components-autocomplete--with-data`

Reply:
Fixed